cluster/network.go

```diff
@@ -0,0 +1,25 @@
+package cluster
+
+import (
+	"fmt"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/rancher/rke/k8s"
+	"github.com/rancher/rke/pki"
+)
+
+func (c *Cluster) DeployNetworkPlugin() error {
+	logrus.Infof("[Network] Setting network plugin: %s", c.NetworkPlugin)
```

Comment:
I thought we we gonna handle addon creation in more generic manner? Imagine the same command will be called on the dns setup as well

Reply:
No, addons will be added in a separate call.